### PR TITLE
docs: add documentation for gradle support

### PR DESCRIPTION
**Pull request by zvigrinberg**

### CODE REVIEW

**Comment by zvigrinberg** [comment]
Hi @agunn303 , can you please review this documentation of adding gradle  support to RHDA Intelli-J plugin?
Thanks!.

File: README.md [hunk 1]
```diff
@@ -19,8 +19,8 @@ Dependency Analytics only accesses your manifest files to analyze your applicati
 vulnerability report.
 
 **IMPORTANT:**
-<br >Currently, Dependency Analytics only supports projects that use Maven (`mvn`), Node (`npm`), Golang (`go mod`) and
-Python (`pip`) ecosystems, and base images in `Dockerfile`.
+<br >Currently, Dependency Analytics only supports projects that use Maven (`mvn`), Node (`npm`), Golang (`go mod`) ,
```
Comment (ritz303) [inline README.md:22]: Towards the end of the line, it looks like there is a space between the last parenthesis bracket and the comma, which there shouldn't be.  The comma goes directly after, like so:

`... Golang (go mod),`

File: README.md [hunk 2]
```diff
@@ -110,6 +110,12 @@ according to your preferences.
   specified in the manifest file will be ignored, and dependency versions will be resolved dynamically instead (this
   feature cannot be enabled when `Strictly match package version` is selected).
 
+- **Gradle** :
+  <br >Set the full path of the Gradle executable, which allows Exhort to locate and execute the `gradle` command to resolve
+  dependencies for Gradle projects.
+  <br >If the path is not provided, your IDE's `PATH` environment will be used to locate the executable.
```
Comment (ritz303) [inline README.md:116]: This sentence is written in passive voice.  Technical documentation should be written in active voice.  I suggest rewriting this sentence in active voice.  For example:

By not setting a path to the `gradle` binary, IntelliJ uses its default path environment to locate the file.

File: README.md [hunk 3]
```diff
@@ -288,8 +301,17 @@ Read our [privacy statement](https://developers.redhat.com/article/tool-data-col
 This plugin respects the settings of the `Telemetry by Red Hat` plugin, which you can learn more
 about [here](https://plugins.jetbrains.com/plugin/16209-telemetry-by-red-hat).
 
+
 ## Support, feedback & questions
 
+The Red Hat Dependency Analytics plugin for IntellJ IDEA in current version, supports the following Intelli-J IDE Versions:
```
Comment (ritz303) [inline README.md:307]: Typo, missing an "i".

`... IntellJ IDEA ...` --> `... IntelliJ IDEA ...`

I don't think there is a dash (-) in "IntelliJ", at least I don't see it that way on their website.  I would remove the dash.  Also, add an "A" to "IDE", and no need to capitalize the word "Versions".

`... Intelli-J IDE Versions ...` --> `... IntelliJ IDEA versions ...`

**Comment by zvigrinberg** [comment]
@agunn303 Thank you!, applied all your suggestions...

**Pull request merged**